### DISCUSSION TITLE
Rendering in a node environment

### DISCUSSION THREAD

**Discussion by foxyblocks** [pixijs/pixijs#8098] [Q&A]
I'm wondering if anyone currently has pixi.js rendering in a node environment. I know that the project https://github.com/Prozi/pixi-shim/ exists and it seems to render pixi in node but I haven't had any success getting it to render images. I have [an issue submitted on that project](https://github.com/Prozi/pixi-shim/issues/23) with code samples of my latest attempt but has anyone else found a way to make this work?

**Comment by Pi-Bouf** [comment]
Hello dude ! It's late, that's true

But you can find some infos here: https://github.com/timknip/node-pixi

Good luck ;)
